src/frontend/Tactics.ml

```diff
@@ -13,6 +13,27 @@ module TB = TermBuilder
 
 open Monad.Notation (RM)
 
+
+let is_total (sign : D.sign) = 
+  let rec go acc = function
+    | D.Field (`User ["fib"],_,D.Clo ([],_)) -> RM.ret @@ acc
```

Comment:
We should probably change the magic `fib` field to be `Machine` at some point. Out of scope for this PR though, so don't worry about it :)